core/src/main/scala/cromwell/core/simpleton/WdlValueBuilder.scala

```diff
@@ -78,6 +78,7 @@ object WdlValueBuilder {
         )                                # End capture.
 
                                   """.trim.r          // The trim is necessary as (?x) must be at the beginning of the regex.
+
```

Comment:
Even more reason to revert this one whitespace line change. :) No git merge conflict.